torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -50,6 +50,7 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processRpc(
 
       // scriptCall is only alive within this block, use reference to avoid copy
       auto& stack = scriptCall.stackRef();
+      RRefContext::getInstance().waitForThreadLocalPendingUsers();
```

Comment:
Should we just do this for all message types and also record for all message types?

Reply:
You mean this this line of code will appear in every switch-case clause? This cannot directly wrap the `deserializeRequest` function, because Python deserialization occurs within the clause. We probably can add a message layer on top of PythonCall to hold the deserialized py::object. Let me try.